packages/gdrcopy.spec

```diff
@@ -98,6 +98,7 @@ rm -rf $RPM_BUILD_DIR/%{name}-%{version}
 
 %files
 %{_prefix}/bin/copybw
+%{_prefix}/bin/copylat
```

Comment:
for updates to the spec, it is common practice to also bump the _release and to add a comment in the changelog

Reply:
similarly in debian/changelog ?

Reply:
please update chengelog section here too